docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
```

Comment:
What does the version relate to? Is it the version of this `docker-compose.yml`?